themes/default/content/what-is/database-comparison-cosmos-db-vs-dynamodb.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Database Comparison: Cosmos DB vs DynamoDB"
+meta_desc: |
+     Compare noSQL databases Cosmos DB vs DynamoDB. See the similarities and differences between these databases to determine which is best for you.
+
+type: what-is
+page_title: "Database Comparison: Cosmos DB vs DynamoDB"
+
+customer_logos:
+  title: Leading engineering organizations are building with Pulumi
+  logos:
+    - items:
+      - snowflake
+      - tableau
+      - atlassian
+      - fauna
+      - ware2go
+    - items:
+      - mindbody
+      - sourcegraph
+      - fenergo
+      - skai
+      - lemonade
+    - items:
+      - clearsale
+      - angellist
+      - webflow
+      - supabase
+      - ro
+---
+
+## Cosmos DB vs DynamoDB: What Are The Similarities?
+
+Both Cosmos DB and DynamoDB are NoSQL, or non-relational, databases. This means their data is stored in some format other than two-dimensional tables. Some commonly-used formats are documents, key-value pairs, graphs, and columns, each with different strengths and tradeoffs.
+
+Cosmos DB and DynamoDB are both highly-available, scalable, globally distributed and fully-managed serverless NoSQL databases. Both function as document based or key-value based databases.
```

Comment:
This is more an editorial thing, but generally style and grammatical guides recommend using an en-dash in compounds that include hyphenated words.
```suggestion
Cosmos DB and DynamoDB are both highly-available, scalable, globally distributed and fully-managed serverless NoSQL databases. Both function as document&ndash; or key-value&ndashbased databases.
```